### DISCUSSION TITLE
v0.0.9 is live 🚀

### DISCUSSION THREAD

**Discussion by AntonOsika** [AntonOsika/gpt-engineer#673] [Announcements]
Great work, everyone working on this release.

I'm personally excited after having used gpt-engineer to improve gpt-engineer itself 👶

We're looking to improve the UX further. Check recent issues labeled for how you can contribute!

## What's New
* 👨‍💻 Add possibility to improve existing codebases  by @leomariga @pbharrin 
* 🧠 Support for open source models, added instructions to docs by @ErikBjare in https://github.com/AntonOsika/gpt-engineer/pull/639
* 🔧  .env setup for OpenAI API key by @OriAshkenazi 
* 🏅 Evals for improve existing code by @pbharrin

## In Detail
* Fixed spelling error on README.md by @salomonj11 in https://github.com/AntonOsika/gpt-engineer/pull/577
* Update TERMS_OF_USE.md by @egecandrsn in https://github.com/AntonOsika/gpt-engineer/pull/575
* Minor fixes to documentation for Windows users by @Oxymoron290 in https://github.com/AntonOsika/gpt-engineer/pull/556
* Docker And Docker-Compose Support by @ayoubfletcher in https://github.com/AntonOsika/gpt-engineer/pull/538
* Remove poetry by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/587
* Refactor files for docker by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/588
* .env setup for OpenAI API key by @OriAshkenazi in https://github.com/AntonOsika/gpt-engineer/pull/499
* Fix missing OPENAI_API_KEY by @pbharrin in https://github.com/AntonOsika/gpt-engineer/pull/592
* Rebase of @leomariga's Existing Project  by @pbharrin in https://github.com/AntonOsika/gpt-engineer/pull/578
* Add possibility to improve existing codebases by @leomariga in https://github.com/AntonOsika/gpt-engineer/pull/465
* Removed unused code in steps.py by @pbharrin in https://github.com/AntonOsika/gpt-engineer/pull/597
* Configure Sweep by @sweep-ai in https://github.com/AntonOsika/gpt-engineer/pull/603
* refactor for clarity by @talkingtoaj in https://github.com/AntonOsika/gpt-engineer/pull/585
* removed file dictionary, and use full path for files in existing code… by @pbharrin in https://github.com/AntonOsika/gpt-engineer/pull/600
* Updated sweep.yaml for pre-commit by @kevinlu1248 in https://github.com/AntonOsika/gpt-engineer/pull/622
* Add numpy-style docstrings to functions by @sweep-ai in https://github.com/AntonOsika/gpt-engineer/pull/615
* Automate Issue and PR Management using GitHub Actions by @sweep-ai in https://github.com/AntonOsika/gpt-engineer/pull/605
* Docker Update by @FancyKat in https://github.com/AntonOsika/gpt-engineer/pull/612
* Fixed Incorrect Github codespaces link by @gssakash in https://github.com/AntonOsika/gpt-engineer/pull/624
* Add benchmark Results for commit 8358b60 with GPT3.5-default by @alvaromat in https://github.com/AntonOsika/gpt-engineer/pull/625
* Add support for Azure OpenAI Service by @niklasfink in https://github.com/AntonOsika/gpt-engineer/pull/640
* feat: added support for open source models, added instructions to docs by @ErikBjare in https://github.com/AntonOsika/gpt-engineer/pull/639
* fix: fix bad merge in #639 by @ErikBjare in https://github.com/AntonOsika/gpt-engineer/pull/644
* Evals for improve existing code by @pbharrin in https://github.com/AntonOsika/gpt-engineer/pull/638
* Merge pull request #1 from AntonOsika/main by @Ax2L in https://github.com/AntonOsika/gpt-engineer/pull/646
* Update readme and docs by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/647
* Improve flow for -i flag by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/652
* First boostrap commit 🚀 Improving usage of typehints. by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/653
* Simplify which folders are used by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/659
* Issue templates by @ATheorell in https://github.com/AntonOsika/gpt-engineer/pull/661
* Revert "Simplify which folders are used" by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/668
* Bump the release version in pyproject.toml by @sweep-ai in https://github.com/AntonOsika/gpt-engineer/pull/666
* Revert "Bump the release version in pyproject.toml" by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/672

## New Contributors
* @salomonj11 made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/577
* @egecandrsn made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/575
* @Oxymoron290 made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/556
* @ayoubfletcher made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/538
* @OriAshkenazi made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/499
* @pbharrin made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/592
* @talkingtoaj made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/585
* @kevinlu1248 made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/622
* @FancyKat made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/612
* @gssakash made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/624
* @alvaromat made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/625
* @niklasfink made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/640
* @ErikBjare made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/639
* @Ax2L made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/646
* @ATheorell made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/661

**Full Changelog**: https://github.com/AntonOsika/gpt-engineer/compare/v0.0.8...v0.0.9

<hr /><em>This discussion was created from the release <a href='https://github.com/AntonOsika/gpt-engineer/releases/tag/v0.0.9'>v0.0.9 is live 🚀</a>.</em>

**Comment by hannahzacharski55** [comment]
Json

**Comment by TamaniPhiri** [comment]
Great stuff
